conbench/app/_plots.py

```diff
@@ -630,28 +630,4 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
     # https://github.com/bokeh/bokeh/pull/2186
     p.xaxis.ticker.desired_num_ticks = 9
 
-    range_tool = bokeh.models.RangeTool(x_range=p.x_range)
-    range_tool.overlay.fill_color = "gainsboro"
-    range_tool.overlay.fill_alpha = 0.2
-
-    select = bokeh.plotting.figure(
-        title="drag the middle and edges of the selection box to change the range above",
```

Comment:
We could do this in a separate PR if you wish, but would you mind enabling the feature of "click/drag on the plot to create a selection range to zoom in to"? I find myself constantly trying to do that on these pages and being disappointed when it pans instead

Reply:
Ack! Let's do this in a separate PR, yes. I also had this 'review/overhaul the Bokeh tools for the history plot' in my mind as wonderful topic for an individual, small PR.